tests/test_bootstrap_wizard.sh

```
@@ -420,26 +420,40 @@ mkdir -p "$empty_path_dir"
+  # `firebase` and `gcloud` are NOT findable — that's the scenario under
+  # test (operator on a fresh machine, picks --firebase none).
+  fb_out=$(PATH="$empty_path_dir:/usr/bin:/bin" \
+           BOOTSTRAP_SKIP_MERGEPATH_GUARD=1 BOOTSTRAP_AUTO_CONFIRM=1 \
+           BOOTSTRAP_AUTO_PROMPT=skip \
+           "$SCRIPT" my-new-repo \
+           --target-dir "$fb_defer_target" \
+           --description d --visibility private --firebase none \
+           --codex-app n --project new --dry-run 2>&1)
+  fb_ec=$?
+  set -e
+  [ "$fb_ec" -eq 0 ] && pass "--firebase none passes even without firebase/gcloud on PATH" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)